Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Entries/SourceAssetBrowserEntry.cpp

```diff
@@ -56,29 +56,17 @@ namespace AzToolsFramework
             }
         }
 
-        void SourceAssetBrowserEntry::Reflect(AZ::ReflectContext* context)
-        {
-            AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
-            if (serializeContext)
-            {
-                serializeContext->Class<SourceAssetBrowserEntry, AssetBrowserEntry>()
-                    ->Version(2)
-                    ->Field("m_sourceId", &SourceAssetBrowserEntry::m_sourceId)
-                    ->Field("m_scanFolderId", &SourceAssetBrowserEntry::m_scanFolderId)
-                    ->Field("m_sourceUuid", &SourceAssetBrowserEntry::m_sourceUuid)
-                    ->Field("m_extension", &SourceAssetBrowserEntry::m_extension);
-            }
-        }
-
         AssetBrowserEntry::AssetEntryType SourceAssetBrowserEntry::GetEntryType() const
         {
             return AssetEntryType::Source;
         }
 
-        const AZStd::string& SourceAssetBrowserEntry::GetExtension() const
+        const AZStd::string SourceAssetBrowserEntry::GetExtension() const
         {
-            return m_extension;
```

Comment:
Another interesting thing of note here, m_extension was present, but never written to or used for anything.  So it was always returning an invalid value.